taildrop/resume.go

```diff
@@ -90,7 +90,7 @@ func (m *Manager) HashPartialFile(id ClientID, baseName string) (next func() (Bl
 	}
 	noopNext := func() (BlockChecksum, error) { return BlockChecksum{}, io.EOF }
 	noopClose := func() error { return nil }
-	if m.opts.DirectFileMode && m.opts.AvoidFinalRename {
+	if m.opts.DirectFileMode {
```

Comment:
This used to depend on `AvoidFinalRename` being true, which was only true for MacOS, right? Should we just remove this block completely?